tinyagentos/routes/mail.py

```
@@ -0,0 +1,230 @@
+    except Exception as exc:
+        return JSONResponse({"error": f"imap error: {exc}"}, status_code=502)
+    return {"folders": folders}
+
+
+@router.get("/api/mail/accounts/{account_id}/messages")
+async def list_messages(
+    request: Request,
+    account_id: str,
+    folder: str = "INBOX",
+    limit: int = 50,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)